apps/src/templates/certificates/GraduateToNextLevel.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import VerticalImageResourceCard from '@cdo/apps/templates/VerticalImageResourceCard';
+import {nextLevelCourses} from '@cdo/apps/templates/certificates/congratsNextLevelActivityCards';
+
+const GraduateToNextLevel = ({nextCourse}) => {
+  // TODO: Create a pattern to make course identifiers are mapped to their titles correctly
+  // look at 'pre-express' and 'accelerated' courses too
```

Comment:
knowing that these strings need to come from the server, are there still question marks in your mind about what this will look like? my understanding is that we only ever show one card in this component, which helps simplify things.

Reply:
Good question. I was originally thinking this would be logic within this component by using the `tutorial` prop (e.g. "coursea-2020") from the Congrats component.

But given that the logic lives on the server, I'm thinking we can pass it in from `/apps/src/sites/studio/pages/congrats/index.js`. I think I might need to pass something in from `congrats_controller.rb`, but I'm not sure that's the best of ideas because we aren't getting the tutorial name from the server––it looks like we're getting it from the params in the URL.

Do you have thoughts on this? I do think we can pass it in from the controller, but we'd need to get the course name first

Reply:
> I think I might need to pass something in from `congrats_controller.rb`, but I'm not sure that's the best of ideas because we aren't getting the tutorial name from the server––it looks like we're getting it from the params in the URL.
> 
> Do you have thoughts on this? I do think we can pass it in from the controller, but we'd need to get the course name first

this sounds like the right idea to me! -- the course name can be extracted from the url param similar to how we do in other controllers: https://github.com/code-dot-org/code-dot-org/blob/f44b633514dc714070101073335fe2040f0a8515/dashboard/app/controllers/certificates_controller.rb#L15-L14

in this case, it is just a base64-encoded string, so you would not need to do the json-decoding step.

Reply:
Oo, that's perfect. Thank you!